ProjectTodo/src/components/Slide.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { Text, View, FlatList } from 'react-native';
+
+export default class Slide extends React.Component {
+  render() {
+    return (
+      <View style={this.props.style.slide1}>
+        <Text>{this.props.title2}</Text>
+        <FlatList
+          extraData={this.props.loader}
+          data={this.props.data}
+          renderItem={({ item }) => (
```

Comment:
Nece da moze ovo ovako :) napisao sam ti na login pr sta bi trebalo da odradis za Flat listu pogledaj to pa mi se javi kada ispravis